operations/mimir/memberlist.libsonnet

```diff
@@ -21,17 +21,30 @@
     memberlist_ring_enabled: false,
 
     // Migrating from consul to memberlist is a multi-step process:
-    // 1) Enable multikv_migration_enabled, with primary=consul, secondary=memberlist, and multikv_mirror_enabled=false, restart components.
+    //
```

Comment:
This documentation is great, but I think it's quite difficult to find for the final user. Can we move it to `docs/sources/operators-guide/deploying-grafana-mimir/jsonnet/`?

Reply:
Yeah, I was thinking the same. I will do that in separate PR, if that's fine.

Reply:
Agree.

Reply:
https://github.com/grafana/mimir/pull/2166